front/src/containers/Islands/WfIslandAggChild.tsx

```diff
@@ -66,19 +95,85 @@ function WfIslandAggChild(props: Props) {
       dispatch(deleteCrowdKeyValueId(props.nctId, key, currentAgg.name))
     }
   }
-  let filteredArray = suggestedLabels?.data?.crowd_keys.filter(x=> x.crowd_key == currentAgg.name);
-  const currentAggBucketsData = filteredArray && filteredArray[0] ?  filteredArray[0].crowd_values : [];
+  let filteredArray = suggestedLabels?.data?.crowd_keys.filter(x => x.crowd_key == currentAgg.name);
+  const currentAggBucketsData = filteredArray && filteredArray[0] ? filteredArray[0].crowd_values : [];
 
   let currentAggBuckets = [];
   //@ts-ignore
   currentAggBucketsData.map(a => currentAggBuckets.push({ "key": a.crowd_value, "docCount": null }));
-  // console.log("🚀 ~ WfIslandAggChild ~ currentAggBuckets!!!!!!!!", currentAggBuckets);
 
   const handleContainerToggle = () => {
     if (aggId) {
       islandConfig[aggId].defaultToOpen = !islandConfig[aggId].defaultToOpen
     }
   }
+
+  const handleEditorText = (e: any) => {
+    setEditorText(e.currentTarget.value);
+    //console.log(editorText)
+  };
+
+  const handleEditSubmit = () => {
+    if (!user || isUpsertingLabel) {
+      !user && dispatch(setShowLoginModal(true))
+      return console.log(!user ? "Sorry, must be logged in to do this" : "Sorry still saving value")
+    }
+    if (crowdKeyValueData.length === 1) {
+      let idToUpdate = crowdKeyValueData[0].id
+      let updatedContent = editorText;
+      dispatch(updateCrowdKeyValueId(idToUpdate, updatedContent, props.nctId))
+      return;
+    }
+    dispatch(insertCrowdKeyValueId(props.nctId, editorText, currentAgg.name, user.id, false, false))
+  };
+
+  if (currentAgg.display === "TEXT_EDITOR") {
+    let editorValue = editorText;
+    if (currentAgg.display === "TEXT_EDITOR" && crowdKeyValueData.length > 0) {
+      // setEditorText(crowdKeyValueData[0].crowd_value)
+      editorValue = crowdKeyValueData[0].crowd_value
+    }
+
+    let configuredLabel = currentAgg?.displayName || '';
+    //const ThemedTitle = isPresearch ? PresearchTitle : ThemedFacetTitle
+
+    return (
+      <Panel>
+        <ThemedPresearchHeader>
+          <PresearchTitle style={{ textAlign: "center" }} >
+
+            {configuredLabel.toUpperCase()}
```

Comment:
would like to discuss why we uppercase and center align here in code vs either controlling in the input of an island config and/or in global styles?